frontend/src/components/FileUpload.spec.cy.js

```diff
@@ -11,35 +10,4 @@ describe('FileUpload', () => {
     cy.get('button');
   });
 
-  it('should fire the beforeUpload when a file is changed', () => {
```

Comment:
this test was providing very little value as-is, but the hooks it's checking have been removed.

The testing coverage for the app needs to be addressed holistically.